README.md

```diff
@@ -47,656 +66,52 @@ rules harder to follow, so please avoid making them.
-of configuration. A single component that takes a complex object as a prop is
-less flexible and more difficult to extend than one that applies a relatively
-simple style and fits together well with other components.
-
-```js
-// bad
-<Accordion
-  data={[
-    {content: 'This is an accordion list item'},
-    {content: 'This is an accordion list item'},
-  ]}
-/>
-
-// good
-<Accordion>
-  <AccordionItem>
-    This is an accordion list item
-  </AccordionItem>
-  <AccordionItem>
-    This is an accordion list item
-  </AccordionItem>
-</Accordion>
-```
-
-### Context or render props
-
-Context and render props are both great patterns for exposing data from the
-innards of a component. Avoid `React.cloneElement` unless there's a good reason,
-because it breaks unpredictably.
-
-```js
-// Connecting `Accordion` to `AccordionItem` with context works fine here, but
-// `cloneElement` will silently pass props to `FancyBorder` instead.
-<Accordion>
-  <FancyBorder>
-    <AccordionItem>This is an accordion list item</AccordionItem>
-  </FancyBorder>
-</Accordion>
-
-// When data needs to be used within a component, render props are a flexible way
-// of enabling this.
-<Form render={({ values, onChange}) =>
-  <FormEntries values={values} onChange={onChange} />
-} />
-
-// Context is good too, but has some awkward ergonomics at times. The below
-// example only works when rendered as a child of our imaginary
-const FormBody = () => {
-  const { values, onChange } = useContext(FormContext)
-
-  return <FormEntries values={values} onChange={onChange} />
-}
-// ...
-<Form><FormBody /></Form>
-```
-
-### Pass props
-
-It's extremely common to want to apply custom styles to an element (margin
-adjustment, etc), so `className` and `style` should be passed through to a
-relevant element. In general, it's most effective to spread all additional
-props, which will pass through event handlers as well.
-
-```js
-// bad
-({ children }) =>
-  <SomeElementEl>{children}</SomeElementEl>
-
-// good
-({ children, ...props }) =>
-  <SomeElementEl {...props}>{children}</SomeElementEl>
-```
-
-### Forward ref
-
-Less commonly, we may need to grab the ref of an element. As a best practice,
-any stylistic component should use `forwardRef` to enable this, and should not
-be written as an arrow function.
-
-```js
-// bad
-({ children, ...props }) => (
-  <SomeElementEl {...props}>{children}</SomeElementEl>
-);
-
-// bad
-// Arrow functions receive their name from the variable they're assigned to,
-// which is broken by wrapping it in `forwardRef`. This leaves the component
-// with no display name.
-React.forwardRef(({ children, ...props }, ref) => (
-  <SomeElementEl ref={ref} {...props}>
-    {children}
-  </SomeElementEl>
-));
-
-// good
-React.forwardRef(function SomeThing({ children, ...props }, ref) {
-  return (
-    <SomeElementEl ref={ref} {...props}>
-      {children}
-    </SomeElementEl>
-  );
-});
-```
+# Process
```

Comment:
I was thinking it'd be nice to have these listed as a baseline, even though we don't follow it exactly as described across all projects